<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1923</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingreso para informes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Llegada a informes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fin atencion informes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Llegada a la puerta desde informes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Llegada a reservas desde informes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Llegada a reservas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingreso para reservas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fin atencion reservas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Llegada a la puerta desde reservas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fin Alarma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio alarma</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="aII">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bII">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aLI">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bLI">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aFI">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bFI">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aLPI">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bLPI">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bLRI">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aLR">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aLRI">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bIR">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bFR">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bLR">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aIR">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aFR">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aLPR">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bLPR">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aIA">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bIA">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_simular">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>90</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Simular :')</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dX/dt = 0.5 * X^2 - (0.2*X) + 5</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaSimulacion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>1511</width>
      <height>721</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Reloj</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo De Evento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiempo Hasta La Proxima Llegada para informes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proxima Llegada de informes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiempo Hasta La Proxima Llegada para reservas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proxima Llegada reservas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proximo Ataque</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duracion del Ataque</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado Informes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Informes ocupado por</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cola Informes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado Reservas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Reservas ocupado por</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cola Reservas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiempo oscioso informes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiempo oscioso reservas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Porcentaje de trabajo alarmas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Atendidos en informes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Atendidos en reservas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad de personas esperando</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad de personas atendidas</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duarcion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duracion">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inicio">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fila de inicio</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaRK">
    <property name="geometry">
     <rect>
      <x>1515</x>
      <y>151</y>
      <width>411</width>
      <height>721</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>xi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>k1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>k2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>k3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>k4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xi+1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yi+1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
